packages/neo-one-server-client/src/npmCheck.ts

```diff
@@ -0,0 +1,74 @@
+import execa from 'execa';
+import * as fs from 'fs-extra';
+
+export interface NPMResult {
+  readonly success: boolean;
+  readonly version: string;
+  readonly date: number;
+}
+const emptyNPMResult: NPMResult = {
+  success: false,
+  version: '',
+  date: 0,
+};
+
+const askNPM = async (pkg: string): Promise<NPMResult> =>
+  new Promise<NPMResult>(async (resolve) => {
+    const infoP = await execa('yarn', ['info', pkg]);
```

Comment:
We shouldn't assume that `yarn` or `npm` in particular are installed. Better to just `fetch` the versions from https://yarnpkg.com/ or https://www.npmjs.com/.

Reply:
https://github.com/neo-one-suite/neo-one/blob/master/packages/neo-one-editor-server/src/resolveDependencies.ts#L85 has an example of fetching the package info and resolving the versions.

Reply:
👍